providers/rxe/rxe_cfg

```diff
@@ -650,7 +650,7 @@ sub main {
     }
 
     # create persistence file if necessary
-    mkdir -p $persistence_path;
+    system("mkdir -p $persistence_path");
```

Comment:
Please do not use the `system` call. Instead use `make_path` which is the corresponding Perl function.